docs/deployment-process/performance.md

```diff
@@ -141,4 +142,8 @@ Steps and actions in your deployment process can be configured to start after th
 
 Similarly to [parallel targets](#parallel-targets), running too many steps in parallel can cause your Octopus Server to become the bottleneck and make your deployments take longer overall.
 
+### Consider how much deployment work the Octopus server is doing {#server-work}
+
+Some steps, like Azure deployments and AWS steps, [run on a worker](docs/administration/workers.md#where-steps-run).  By default, thats the [built-in worker](/docs/administration/workers#built-in-worker) in the Octopus server.  That means the step invokes a (or many) Calamari processes on the server machine to do the deployment work.  That workload can be shifted off the server and onto [workers](/docs/infrastructure/workers.md).  See this [blog post](https://octopus.com/blog/workers-performance) for a way to begin looking at workers for performance.
```

Comment:
thats -> that's
a (or many) -> could invoke one or more Calamari processes
That workload -> These workloads?

Reply:
> That workload can be shifted off the server and onto [workers]
That should be either: 
That workload can be moved off the server onto a [Worker]...
or:
These workloads can be moved off the server onto [Workers]...